aii-core/src/test/perl/grub2_efi_options.t

```diff
@@ -0,0 +1,72 @@
+use strict;
+use warnings;
+use CAF::Application;
+use Test::Quattor;
+use Test::More;
+use NCM::Component::PXELINUX::constants qw(:pxe_constants);
+use AII::Shellfe;
+use Readonly;
+
+Readonly my $AII_CONFIG_EMPTY => 'src/test/resources/aii-empty.conf';
+Readonly my $AII_CONFIG_NBPDIR => 'src/test/resources/aii-nbpdir.conf';
+Readonly my $AII_CONFIG_GRUB2_ONLY => 'src/test/resources/aii-grub2_only.conf';
+Readonly my $AII_CONFIG_PXELINUX_GRUB2 => 'src/test/resources/aii-pxelinux_grub2.conf';
+Readonly my $AII_CONFIG_PXELINUX_ONLY => 'src/test/resources/aii-pxelinux_only.conf';
+Readonly my $AII_LOG_FILE => 'target/test/aii-shellfe.log';
```

Comment:
you are using a log file in a directory that you do not create in this test. so if this test runs first, this will fail.
add
```perl
mkdir 'target/test';
```
in the start of each test that uses something similar.

but i would like to point out that it is better to use a filename per test, so if you need to debug, the log file only has data from one test.